src/main/java/gregtech/integration/hwyla/provider/RecipeLogicDataProvider.java

```diff
@@ -64,16 +64,15 @@ public List<String> getWailaBody(ItemStack itemStack, List<String> tooltip, IWai
         if (accessor.getNBTData().hasKey("gregtech.AbstractRecipeLogic")) {
             NBTTagCompound tag = accessor.getNBTData().getCompoundTag("gregtech.AbstractRecipeLogic");
             if (tag.getBoolean("Working")) {
-                int EUt = tag.getInteger("RecipeEUt");
-                int absEUt = Math.abs(EUt);
-                boolean consumer = EUt > 0;
+                int eut = tag.getInteger("RecipeEUt");
```

Comment:
This also needs a check now for `tag.getLong()` correct? If you are starting to serialize EUt as a long, you should also start reading it as a long.